api/src/main/java/javax/jdo/spi/JDOImplHelper.java

```diff
@@ -1065,7 +1065,7 @@ public void nonBinaryCompatibleMakeDirty(Object pc, String fieldName) {
       try {
         if (si.makeDirty(pc, fieldName)) return;
       } catch (Throwable t) {
```

Comment:
Out of scope for this PR but maybe a case for a separate PR?
Maybe we should not catch/ignore `Throwable` here, that would include `OutOfMemoryException` and such.

Reply:
Please file a separate JIRA for this. Then we can discuss which exceptions should be swallowed.

Reply:
Done, see https://issues.apache.org/jira/browse/JDO-824.